src/lib/react/index.ts

```diff
@@ -104,6 +104,18 @@ export const reactUniDriver = (containerOrFn: ElementOrElementFinder): UniDriver
 				Simulate.click(el, {button: 0});
 			}
 		},
+		hover: async () => {
```

Comment:
I was wondering as well. However, the reason I decided to implement it by "hovering" is because the way consumers will probably implement in most cases.
As far as I understand, the main use-case for hovering will be implemented by using css (selector:hover). Therefore, although mouseEnter/mouseOver / mouseLeave is more accurate - and even provides more functionality - I felt like it will be more user-friendly to use hover.

Nevertheless, I am obviously open to the alternatives.

Reply:
In cases of tooltip usage for example, it makes sense to do a mouseEnter function to trigger an open event and mouseOut to close.
So it might make sense to have both the hover abstraction and the lower level api

Reply:
what if `hover` performs both mouseEnter and mouseOver? 
A user will only "move mouse to" which is "hover" It feels close to #13 in that sense.

However, the same concern in #13 is back - the mouse is global, meaning that ideally `hover` will trigger a mouseLeave on wherever the mouse was beforehand (that is also true to click).

I think it's safe to start with both events on the DOM (react) adapter, and assuming that on webdriver-ish apis it'll just work

thoughts?

Reply:
Seems to me like a good suggestion. I'll implement.

Reply:
👍